src/command_modules/azure-cli-security/azure/cli/command_modules/security/_validators.py

```diff
@@ -0,0 +1,16 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from msrestazure.tools import is_valid_resource_id
+from knack.util import CLIError
+
+
+def validate_resource(cmd, namespace):  # pylint: disable=unused-argument
```

Comment:
Is this being used anywhere?

Reply:
updated the validators file with used validations